packages/ai/src/providers/anthropic/index.ts

```diff
@@ -130,7 +131,7 @@ const createChatCompletion: ICreateChatCompletion = async (body) => {
     executionDone();
     throw new AIError(
       AIErrorEnum.FAILED_CHAT_COMPLETION,
-      `failed chat completion for model ${body.model} with message ${(e as Error).message} `,
+      `failed chat completion for model ${body.model} with message ${(e as Error).message}`,
```

Comment:
let's ensure if timeout is hit, the logging/messaging communicates to the user what has happened - so that they can increase the timeout if required